user-guides/staking-pools.md

```diff
@@ -0,0 +1,32 @@
+---
+description: How to stake agTokens or sanTokens to get ANGLE rewards using Angle
+  app
+---
+
+# Staking Pools
+
+You can stake your sanTokens and agTokens in the staking page to earn ANGLE rewards.
+
+The weights of rewards distribution between pools is updated weekly by governance. You can find more information about the tokenomics [here in the docs](../governance/angle-token/README.md), or in [this blog post](https://blog.angle.money/angle-protocol-tokenomics-29ea8b7bf001).
+
+On the main Staking page, you can choose to `Stake` your tokens, `Claim` your rewards, or `Unstake` your tokens (which will claim rewards simultaneously if you unstake everything you staked).
+
+You need to have either agTokens or sanTokens to be able to stake withing Angle.
+
+## Stake
```

Comment:
Add information about the fact that perpetuals automatically accumulate ANGLE tokens (here and in the HA page)